src/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import * as styles from "./Tabs.module.css";
+import { Tab, Tabs, TabList, TabPanel } from "react-tabs";
+import { BadgeCounter } from "@utrecht/component-library-react";
+
+interface testProps {
+  tabs: ITabProps[];
+  tabContent: ITabPanelProps[];

```

Comment:
I think by requesting the required props like this, the implementation can break (maybe not actually error, but definitely have some unintended implementations).

Right now, we're required to maintain two arrays that are synced with each other in the implementation layer.

E.g. (simplified): 

- Tabs: `tabs: ["Tab one", "Tab two", "Tab three"];`
- Tab content: `tabContent: `["Tab one content", "Tab two content"]`;

This implementation would be accepted, but Tab three doesn't actually have any content to show for it. Also:

- Tab content: `tabContent: `["Tab one content", "Tab *three* content"];`;

Also an valid input, but now the tab three content is shown in tab two, whereas tab three doesn't seem to have content. 

*Conclusion:*

I think we'd be better off making _one_ array which holds the Tab and TabContent in the same object, something like this (dummy code):

```
type TTab {
  title: string;
  titleBadge?: string;

  content: JSX.Element;
}

interface TabProps {
  tabs: TTab[];
}

export const Tabs ({tabs}) ... {
  // map over tabs.title to create tab headers
  // map over tabs.content to create the panels
}
```